ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Branch Android SDK change log
 
+- v5.6.0
+  * _*Master Release*_ - Jun 28, 2023
+    - Added new method for logging a Branch Event with a callback, `logEvent(Context context, final BranchLogEventCallback callback)`
```

Comment:
Add a line for: Add proguard consumer rules to prevent R8 warning on Gradle 8 builds